.travis.yml

```diff
@@ -1,24 +1,50 @@
+version: ~> 1.0
 language: scala
 
-before_install: curl -Ls https://git.io/jabba | bash && . ~/.jabba/jabba.sh
+before_install: curl -Ls https://raw.githubusercontent.com/shyiko/jabba/0.11.2/install.sh | bash && . ~/.jabba/jabba.sh
```

Comment:
Maybe use the version with retries we're adopting in other places? (https://github.com/akka/akka-projection/pull/289/files, https://github.com/lagom/lagom/pull/2908,...)

```
 - if [ ! -f ~/.jabba/jabba.sh ]; then curl -L -v --retry 5 -o jabba-install.sh https://raw.githubusercontent.com/shyiko/jabba/0.11.2/install.sh && bash jabba-install.sh; fi
  - . ~/.jabba/jabba.sh
```